display/ssd1306.h

```diff
@@ -468,26 +486,63 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, Sa
 	  ShowFile(&IMG_idle, 3600000.0);
 	}
 	return;
-      case EFFECT_BLAST:
-	ShowFile(&IMG_blst, font_config.ProffieOSBlastImageDuration);
-	return;
-      case EFFECT_CLASH:
-	ShowFile(&IMG_clsh, font_config.ProffieOSClashImageDuration);
-	return;
-      case EFFECT_FORCE:
-	ShowFile(&IMG_force, font_config.ProffieOSForceImageDuration);
-	return;
       case EFFECT_LOCKUP_BEGIN:
-	ShowDefault();
-	return;
+      	ShowDefault();
+      	return;
       case EFFECT_LOCKUP_END:
-	ShowDefault(true);
-	break;
+      	ShowDefault(true);
+      	break;
+
+      default: break;
+    }
+  }
+
 
+ void SB_Effect2(EffectType effect, float location) override {
```

Comment:
I'm not sure why this was moved to SB_Effect2, but it does mean that we can know the sound lengths that went with these effects, and we could potentially use that to set the durations below...

Reply:
All of this was an incremental step. I have further changes pending, but you asked for just the effects like this as a baby step towards the end goal; to just have all of them follow the wav duration (if #define OLED_SYNCED_EFFECTS or not).
I will revisit all this ASAP.

Reply:
That's all fine, but it doesn't make sense to add a new variable for how long to show the preon image.
If you don't want to add the code to make it depend on the wav, then just use the fontImageDuration so you don't have to add a new variable that doesn't really make sense.

Reply:
>  If you don't want to add the code to make it depend on the wav, 

I do. I think that's a choice way to handle these, auto timed to the effect itself, no?
Again, trying to get some time. Will need to revisit this and submit what I have working/happy on my end.